docs/blog/newsletter-february-2020.rst

```diff
@@ -0,0 +1,96 @@
+#learn-tech-writing book club
+-----------------------------
+
+There's been a buzz of activity all through January in the `#learn-tech-writing <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ channel, where folks have been discussing *The Product is Docs* chapter-by-chapter. As those involved have kindly started an effort to write up all the fantastic conversations on the topic, we're not going to write it up in the newsletter, but we wanted to point you towards what's been written up so far:
+
+- `Agile and technical writing <https://github.com/writethedocs/www/blob/bookClubWriteUps/docs/book-club/splunk-product-docs/2-agile.rst>`__
+- `Documenting third-party products <https://github.com/writethedocs/www/commit/89ead30998101c1ce1038203c74e53aa50719a29>`__
+- `Technical editing <https://github.com/writethedocs/www/blob/bookClubWriteUps/docs/book-club/splunk-product-docs/13-technical-editing.rst>`__
+
+If you'd like to help out with write-ups, drop into the channel!  As a bonus, they're looking into what to discuss next, so if you have a book you think would be great for group discussions, go ahead and make the suggestion. 
+
+---------------------------------------------
+Supporting documentation, documenting support
+---------------------------------------------
+
+A recent question about how to balance docs work and support content produced some interesting responses. The original post assumed two distinct teams, one for documentation and one for content produced by a support team, with each team producing distinct content. This is evidently the situation in many companies. For example, one poster described a workflow where answers to frequently-asked support questions are added to the documentation, but if a potential docs page seems particularly complicated, or addresses an edge case, the writer asks support to document it. This story describes distinct teams -- Docs and Support -- and distinct types of content.
+
+But many replies to the original post were from people who work on support teams, not on dedicated docs teams. And they write the docs. In their situations, there isn't a clear distinction between roles, nor between types of content. More than one reply described scenarios in which all docs are produced by support. 
+
+Still other replies came from technical writers whose role emerged from support. Some writers report to a support manager, and some people in a dedicated writer role work from support tickets, to make sure the docs address known customer issues. Some folks work on both product docs and separate knowledgebase articles.
+
+In short, there are myriad ways to organize the relationship between docs teams and support teams, and between the different (or not-so-different) types of content. What everyone seemed to agree on, though, was the need for close and well-coordinated communication across teams and deliverables.
+
+--------------------------------------
+How to set priorities in documentation
```

Comment:
"How to set priorities on a documentation team" ? Seems like prioritizing isn't a team-only issue, so title here could be clearer ...

Reply:
"How to set Documentation priorities"? :-p

Reply:
fixed!